app/views/artefacts/form/_tags.html.erb

```diff
@@ -1,56 +1,5 @@
 <%- unless f.object.app_without_tagging_support? %>
   <%- if f.object.tagging_migrated? %>
```

Comment:
All the things will be "migrated" after this, so this conditional can be removed.